e2e/scientists_queries.yaml

```diff
@@ -1120,7 +1120,7 @@ queries:
     checks:
       - num_cols: 2
       - num_rows: 11
-      - selected: [ "?x", "?t", "?ql_textscore_t" ]
+      - selected: [ "?x", "?ql_textscore_t", "?t" ]
```

Comment:
Still something failing, the score column also contributes to the result (there is a NULL value or something to ignore the value of a column.